packages/manager/modules/netapp/src/dashboard/volumes/dashboard/snapshots/delete/template.html

```diff
@@ -0,0 +1,34 @@
+<form
+    id="deleteSnapshot"
+    name="deleteSnapshot"
+    data-ng-submit="deleteSnapshot.$valid && $ctrl.deleteSnapshot()"
+    novalidate
+>
+    <oui-modal
+        data-heading="{{ ::'netapp_volumes_snapshots_delete_title' | translate }}"
+        data-primary-label="{{ ::'netapp_volumes_snapshots_delete_confirm' | translate }}"
+        data-secondary-label="{{ ::'netapp_volumes_snapshots_delete_cancel' | translate }}"
+        data-secondary-action="$ctrl.goBack()"
+        data-on-dismiss="$ctrl.goBack()"
+        data-loading="$ctrl.isDeleting"
+        type="warning"
```

Comment:
`data-` prefixes could be removed no ? (or add the missing one for `type`attribute)